src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -146,6 +167,12 @@ public function build() {
 
     $sort_options = $backend->getSortOptions();
 
+    $bs_version = 4;
+    $theme_name = $this->themeManager->getActiveTheme()->getName();
```

Comment:
There might be sites using the custom theme based on the rose / lily / carnation. Is there a way to check for the "base theme" here if this is the case?
And then there also might be sites using completely custom theme...

Reply:
I even think about introducing the actual setting somewhere is admin form to have ability to set it there... out of scope for now.

Reply:
For now, we get a base theme or if it missed an active theme and set related BS version. 
But the point with a separate field for it sounds cool.